verticaltabs.jsm

```diff
@@ -101,19 +101,25 @@ function VerticalTabs(window, {newPayload, addPingStats, AppConstants, setDefaul
 VerticalTabs.prototype = {
 
   init: function () {
-    this.BrowserOpenTab = this.window.BrowserOpenTab;
-    this.window.BrowserOpenTab = function () {
-      this.pushToTop = true;
-      this.window.openUILinkIn(this.window.BROWSER_NEW_TAB_URL, 'tab');
-      this.pushToTop = false;
```

Comment:
So, I'm a little confused as to why we don't need to set `pushToTop` back to `false` anymore…
So, I tried removing the `pushToTop` variable entirely, and just using `Services.prefs.getBoolPref('extensions.verticaltabs.opentabstop')` on line 465, and it seems to work okay (in Firefox Release), so maybe we want to do that instead?
(We still seem to need the `BrowserOpenTab` function for some reason, though.)